jwst/datamodels/ifuimage.py

```diff
@@ -0,0 +1,93 @@
+from . import image
+from . import model_base
+
+__all__ = ['IFUImageModel']
+
+
+#class IFUImageModel(image.ImageModel):
+class IFUImageModel(model_base.DataModel):
+    """
+    A data model for 2D IFU images.
+
+    Parameters
+    ----------
+    init : any
+        Any of the initializers supported by `~jwst.datamodels.DataModel`.
+
+    data : numpy array
+        The science data.
+
+    dq : numpy array
+        The data quality array.
+
+    err : numpy array
+        The error array.
+
+    relsens : numpy array
+        The relative sensitivity table.
+
+    relsens2d: numpy array
+        The relative sensitivty 2D array.
+    """
+    schema_url = "ifuimage.schema.yaml"
+
+    def __init__(self, init=None, data=None, dq=None, err=None, relsens=None,
+                 relsens2d=None, zeroframe=None, area=None,
+                 pathloss_uniformsource=None, pathloss_pointsource=None,
+                 wavelength_pointsource=None, wavelength_uniformsource=None,
+                 **kwargs):
+        if isinstance(init, image.ImageModel):
+            super(IFUImageModel, self).__init__(init=None, **kwargs)
+            self.update(init)
+            self.data = init.data
+            self.dq = init.dq
+            self.err = init.err
+            if hasattr(init, 'relsens'):
+                self.relsens = init.relsens
+            if hasattr(init, 'area'):
+                self.area = init.area
+            if hasattr(init, 'relsens2d'):
+                self.relsens2d = init.relsens2d
+            if hasattr(init, 'var_poisson'):
+                self.var_poisson = init.var_poisson
+            if hasattr(init, 'var_rnoise'):
+                self.var_rnoise = init.var_rnoise
+            return
+        super(IFUImageModel, self).__init__(init=init, **kwargs)
+
+        if data is not None:
+            self.data = data
+
+        if dq is not None:
+            self.dq = dq
+
+        if err is not None:
+            self.err = err
+
+        if relsens is not None:
```

Comment:
A check of the photom step indicates that the RELSENS extension is NOT populated for either NIRSpec IFU or MIRI MRS exposures. They only use the RELSENS2D extension. Hence RELSENS should not be needed in IFUImageModel.